### PR TITLE
multi: faraday rename

### PR DESCRIPTION
**Pull request by carlaKC**
Rename le project of many names to faraday. 
Changes isolated to a single commit, just search-and-replaced everything + regenerated protos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# governator
+# faraday
 
-[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/lightninglabs/governator/blob/master/LICENSE)
+[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/lightninglabs/faraday/blob/master/LICENSE)
 
-The governator is an external service intended to be run in conjunction with the [lnd](https://github.com/lightningnetwork/lnd) implementation of the [Lightning Network](https://lightning.network). It queries LND for information about its existing channels and provides channel close recommendations if channels are under-performing.  
+The faraday is an external service intended to be run in conjunction with the [lnd](https://github.com/lightningnetwork/lnd) implementation of the [Lightning Network](https://lightning.network). It queries LND for information about its existing channels and provides channel close recommendations if channels are under-performing.
```
Comment (cfromknecht) [inline README.md]: "The" no longer necessary?

File: README.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/lightninglabs/faraday/blob/master/LICENSE)
 
-The governator is an external service intended to be run in conjunction with the [lnd](https://github.com/lightningnetwork/lnd) implementation of the [Lightning Network](https://lightning.network). It queries LND for information about its existing channels and provides channel close recommendations if channels are under-performing.  
+The faraday is an external service intended to be run in conjunction with the [lnd](https://github.com/lightningnetwork/lnd) implementation of the [Lightning Network](https://lightning.network). It queries LND for information about its existing channels and provides channel close recommendations if channels are under-performing.  
 
 ## Installation
-A [Makefile](https://github.com/lightninglabs/governator/blob/master/Makefile) is provided. To install governator and al its dependencies, run:
+A [Makefile](https://github.com/lightninglabs/faraday/blob/master/Makefile) is provided. To install faraday and al its dependencies, run:
```
Comment (cfromknecht) [inline README.md]: al -> all

File: frdrpc/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 gvrpc
```
Comment (guggero) [inline frdrpc/README.md]: s/gvrpc/frdrpc/

File: frdrpc/rpcserver.go [hunk 4]
```diff
@@ -1,14 +1,14 @@
 // Package gvrpc contains the proto files, generated code and server logic
```
Comment (guggero) [inline frdrpc/rpcserver.go]: s/gvrpc/frdrpc

**Pull request merged**